pkg/build/controller/factory/factory.go

```diff
@@ -103,11 +103,11 @@ func (factory *BuildControllerFactory) Create() controller.RunnableController {
 					if len(build.Status.Reason) == 0 {
 						build.Status.Reason = buildapi.StatusReasonError
 					}
-					build.Status.Message = msg
+					build.Status.Message = formatError(msg)
```

Comment:
it wont let me comment on that line, but isnt this breaking this check `build.Status.Message != msg`  since msg is the unformatted version but build.Status.Message is the formatted verison?